.spelling

```diff
@@ -239,6 +243,7 @@ configurability
 conformant
 CONNECT-IP
 CONNECT-UDP
+conntrack
```

Comment:
I looks like you added backticks around the instance in the doc, so may not need this one anymore.